docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -122,16 +122,47 @@ The `InitConfiguration` resource provides two more parameters for non-standard t
 
-The following parameters must be set if the [Nexus](https://github.com/sonatype/nexus-public) repository manager is used:
+##### Requirements
 
-* Enable `Docker Bearer Token Realm`:
-  ![Enable `Docker Bearer Token Realm`](images/registry/nexus/nexus1.png)
+The following requirements must be met if the [Nexus](https://github.com/sonatype/nexus-public) repository manager is used:
+
+* `Docker Bearer Token Realm` is enabled
+* Docker proxy repository was created
+* `Allow anonymous docker pull` is enabled
+* Access control was configured:
+  * Nexus role with `nx-repository-view-docker-<your-repo>-browse` and `nx-repository-view-docker-<your-repo>-read` permissions was created
+  * Nexus user was created with a role above
+* `Maximum metadata age` for created repository was set to 0
 
-* Enable anonymous registry access (otherwise, Bearer authentication [won't work](https://help.sonatype.com/repomanager3/system-configuration/user-authentication#UserAuthentication-security-realms)):
-  ![Enable anonymous registry access](images/registry/nexus/nexus2.png)
+##### How to configure Nexus
 
-* Set the `Maximum metadata age` to 0 (otherwise, the automatic update of Deckhouse will fail due to caching):
-  ![Set the `Maximum metadata age`](images/registry/nexus/nexus3.png)
+* Enable `Docker Bearer Token Realm`:
+  ![Enable `Docker Bearer Token Realm`](images/registry/nexus/nexus-realm.png)
+
+* Create docker proxy repository pointing to [Deckhouse registry](https://registry.deckhouse.io/):
+  ![Create docker proxy repository](images/registry/nexus/nexus-repository.png)
+  
+* Fill in the create page fields as follows:
+  * `Name` should contain created repository name, e.g. `d8-proxy`
+  * `Repository Connectors / HTTP` or `Repository Connectors / HTTPS` should contain dedicated port for created repository, e.g. `8123` or other
+  * `Allow anonymous docker pull` should be enabled in order for Bearer-token authentication to [work](https://help.sonatype.com/repomanager3/system-configuration/user-authentication#UserAuthentication-security-realms), however anonymous access [won't be working](https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/docker-registry/docker-authentication#DockerAuthentication-UnauthenticatedAccesstoDockerRepositories) unless it was explicitly enabled in Settings -> Security -> Anonymous Access and `anonymous` user was granted access rights to created repository
+  * `Remote storage` should be set to `https://registry.deckhouse.io/`
+  * `Auto blocking enabled` and `Not found cache enabled` could be disabled for debugging purposes, otherwise it should be enabled
+  * `Maximum Metadata Age` should be set to 0
+  * `Authentication` should be enabled if you are planning to use Enterprise Edition with the following fields set as:
+    * `Authentication Type` should be set to `Username`
+    * `Username` should be set to `license-token`
+    * `Password` should be set to your licence key for Deckhouse EE
```

Comment:
```suggestion
  * `Name` should contain created repository name, e.g. `d8-proxy`.
  * `Repository Connectors / HTTP` or `Repository Connectors / HTTPS` should contain a dedicated port for the created repository, e.g. `8123` or another.
  * `Allow anonymous docker pull` should be enabled in order for Bearer-token authentication to [work](https://help.sonatype.com/repomanager3/system-configuration/user-authentication#UserAuthentication-security-realms), however anonymous access [won't be working](https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/docker-registry/docker-authentication#DockerAuthentication-UnauthenticatedAccesstoDockerRepositories) unless it was explicitly enabled in Settings -> Security -> Anonymous Access and `anonymous` user was granted access rights to the created repository.
  * `Remote storage` should be set to `https://registry.deckhouse.io/`.
  * `Auto blocking enabled` and `Not found cache enabled` could be disabled for debugging purposes, otherwise it should be enabled.
  * `Maximum Metadata Age` should be set to 0.
  * `Authentication` should be enabled if you are planning to use Deckhouse Enterprise Edition with the following fields set:
    * `Authentication Type` should be set to `Username`.
    * `Username` should be set to `license-token`.
    * `Password` should be set to your license key for Deckhouse Enterprise Edition.
```